packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -336,6 +339,15 @@ export class RunningSummarizer implements IDisposable {
                     timePending: Date.now() - this.heuristics.lastAttempted.summaryTime,
                 });
             });
+        // back-compat 0.34 noSetPendingAckTimerTimeoutCallback
+        if (summaryCollection.setPendingAckTimerTimeoutCallback !== undefined) {
+            summaryCollection.setPendingAckTimerTimeoutCallback(maxAckWaitTime, () => {
+                if (this.pendingAckTimer.hasTimer) {
+                    this.pendingAckTimer.clear();
+                    this.trySummarize("summaryAckMiss");
```

Comment:
I don't think you need to (and probably shouldn't) try to summarize here. In both cases, I think it makes more sense to let the heuristics play out naturally:
1. During the initial startup we run `waitStart()`, and when the timer completes, it should go ahead and continue to initializing the heuristics and then only summarizing if any ops have come in, which they will have.
2. While waiting for a summary broadcast op or ack/nack, we will encounter a "tryWhileSummarizing" case and try to summarize on our own anyway.

Reply:
Sounds good. Removed it.